src/snowflake/snowpark/table_function.py

```diff
@@ -78,6 +78,7 @@ def over(
         Note that if this function is called but both ``partition_by`` and ``order_by`` are ``None``, the table function call will put all input rows into a single partition.
         If this function isn't called at all, the Snowflake database will use implicit partitioning.
         """
+        # TODO(oplaton): Extend over() to collect the AST.
```

Comment:
JIRA ticket?

Reply:
this may be similar to window functions, I think I implemented there some logic to help the over() case.

Reply:
Thanks for the pointer! Will examine the window `over()` implementation. Will file JIRA ticket out of band.